### DISCUSSION TITLE
LCE install RPi4

### DISCUSSION THREAD

**Discussion by ltforsman** [larq/compute-engine#707] [Q&A]
Hello, when trying to simply install larq-compute-engine using 'pip install larq-compute-engine' I get the error:

ERROR: Could not find a version that satisfies the requirement larq-compute-engine (from versions: none)
ERROR: No matching distribution found for larq-compute-engine

I am working on a RPi 4 running Ubuntu 20.04. Any suggestions for how to install properly? Thanks

**Comment by CNugteren** [comment]
Unfortunately we do not provide LCE converter Python packages for non-x86 Linux systems such as your aarch64 RaspberryPi 4. If you can, try to install the pip package on an x86-64 system with macOS, Linux, or Windows and run the converter from there. Then copy the resulting `.tflite` file over to your RaspberryPi and proceed with running the compute-engine there.

**Reply by ltforsman** [reply]
Does that mean we should be able to benchmark on intel x86 cpu's as well?

**Reply by CNugteren** [reply]
No, not really. Although it might work, the Larq Compute Engine only has optimized code for ARM CPUs (e.g. Cortex-A). Thus, if you run it on an x86 CPU it might be slow since it runs with reference non-optimized code.
